plugins/woocommerce-admin/client/customize-store/design-with-ai/services.ts

```diff
@@ -371,16 +387,6 @@ export const assembleSite = async (
 			error: error instanceof Error ? error.message : 'unknown',
 		} );
 	}
-
-	// @ts-ignore No types for this exist yet.
-	const { invalidateResolutionForStoreSelector } = dispatch( coreStore );
-
-	// Invalid the selectors so that the new template/style are used in assembler hub.
-	invalidateResolutionForStoreSelector( 'getEntityRecord' );
-	invalidateResolutionForStoreSelector(
-		'__experimentalGetCurrentGlobalStylesId'
-	);
-	invalidateResolutionForStoreSelector( '__experimentalGetTemplateForLink' );
```

Comment:
I've tried calling `invalidateResolutionForStoreSelector` and even using `invalidateResolutionForStore` but still seeing the old content, so decided to do full redirection instead.